src/openrct2-ui/windows/TileInspector.cpp

```diff
@@ -558,6 +562,14 @@ static void window_tile_inspector_select_element_from_list(rct_window* w, int32_
     else
     {
         windowTileInspectorSelectedIndex = index;
+
+        // update the brake speed to current one
```

Comment:
```suggestion
        // Update the brake speed to the current one
```

Reply:
This should be removed entirely once you get rid of the global variable `windowTileInspectorTrackSpeed` though.